docs/plugins.asciidoc

```diff
@@ -6,6 +6,14 @@
 Add-on functionality for Kibana is implemented with plug-in modules. You can use the `bin/kibana-plugin`
 command to manage these modules. You can also install a plugin manually by moving the plugin file to the
 `plugins` directory and unpacking the plugin files into a new directory.
+
+[IMPORTANT]
+.Plugin compatibility
+==============================================
+
+The Kibana plugin interfaces are in a state of constant development.  We cannot provide backwards compatibility for plugins due to the high rate of change.  Kibana enforces that the installed plugins match the version of Kibana itself.  Plugin developers will have to release a new version of their plugin for each new Kibana release as a result.
+
+==============================================
 --
```

Comment:
You need to delete the + symbols from the beginning of each line. Other than that, LGTM.